src/programming-assistants.bib

```diff
@@ -0,0 +1,68 @@
+@inproceedings{gvero2015synthesizing,
+  title={Synthesizing Java expressions from free-form queries},
+  author={Gvero, Tihomir and Kuncak, Viktor},
+  booktitle={Acm Sigplan Notices},
```

Comment:
don't cite SIGPLAN notices (it just republishes conference papers), cite the original conference